### PR TITLE
Use new taint key in design doc

### PR DESCRIPTION
**Pull request by resouer**
Part of: https://github.com/kubernetes/kubernetes/issues/51246

~~**DO NOT MERGE** until kubernetes/kubernetes#51266 is in!~~

### CODE REVIEW

**Comment by resouer** [comment]
Will merge this one into https://github.com/kubernetes/community/pull/1043

**Comment by resouer** [comment]
cc @k82cn for review

File: contributors/design-proposals/scheduling/taint-node-by-condition.md [hunk 1]
```diff
@@ -18,20 +18,20 @@ In addition to this, with taint-based-eviction, the Node Controller already tain
 | ConditionType      | Condition Status   |Effect        | Key      |
 | ------------------ | ------------------ | ------------ | -------- |
 |Ready               |True                | -            | |
-|                    |False               | NoExecute    | node.kubernetes.io/notReady           |
-|                    |Unknown             | NoExecute    | node.kubernetes.io/unreachable        |
-|OutOfDisk           |True                | NoSchedule   | node.kubernetes.io/outOfDisk          |
+|                    |False               | NoExecute    | node.kubernetes.io/not-ready           |
+|                    |Unknown             | NoExecute    | node.alpha.kubernetes.io/unreachable        |
```
Comment (k82cn) [inline contributors/design-proposals/scheduling/taint-node-by-condition.md]: why not also remove alpha for `unreachable`? so we will handle `not-ready` and `unreachable` in the same way :).

**Comment by resouer** [comment]
`not-ready` is fixed in 1.8, and rest of them are fixed in 1.9 due to wider discussion.

File: contributors/design-proposals/scheduling/taint-node-by-condition.md (hunk 1 above)
Comment (resouer) [reply contributors/design-proposals/scheduling/taint-node-by-condition.md]: Yes, we should, bad rebase.

**Comment by resouer** [comment]
Addressed comments.

> just can not remember why we only change not-ready :).

Not clear for me either ...

File: contributors/design-proposals/scheduling/taint-node-by-condition.md (hunk 1 above)
Comment (resouer) [reply contributors/design-proposals/scheduling/taint-node-by-condition.md]: Fired up https://github.com/kubernetes/kubernetes/issues/54198 to track this inconsistency

Comment (k82cn) [reply contributors/design-proposals/scheduling/taint-node-by-condition.md]: great！

**Comment by resouer** [comment]
@davidopp  thanks, merge?

**Comment by resouer** [comment]
OK. The bot is clever than I expect :)

**Pull request merged**